### PR TITLE
Create migration guidance for `high-contrast-outline()` and `high-contrast-border()` mixins

### LINKED ISSUE #5043: Create migration guidance for `high-contrast-outline()` and `high-contrast-border()` mixins

### PR DESCRIPTION
**Pull request by lgriffee**
### WHY are these changes introduced?

Fixes #5043

### WHAT is this pull request doing?

- Adding documentation for the`high-contrast-outline()` mixin to our migration guide
- Adding documentation for the`high-contrast-outline()` mixin to our migration guide

<hr>

### Helpful References

- #4962
- #4900

### CODE REVIEW

File: documentation/guides/migrating-from-v8-to-v9.md [hunk 1]
```diff
@@ -89,6 +89,8 @@ To help you quickly add these functions and mixins back to your repo, we've crea
 
 A list of functions/mixins and their value equivalents or new token values.
 
+#### `border-width()`
+
```
Comment (alex-page) [inline documentation/guides/migrating-from-v8-to-v9.md:93]: Should this be removed and added to another PR?

File: documentation/guides/migrating-from-v8-to-v9.md [hunk 2]
```diff
@@ -178,6 +180,28 @@ We replaced a few of the following filter function instances with color tokens i
...
 | `filter('action')`<br>`filter('action', 'base')` | `brightness(0) saturate(100%) invert(20%) sepia(59%) saturate(5557%) hue-rotate(162deg) brightness(95%) contrast(101%)`  |
 
+#### `high-contrast-outline()`
+
+| Mixin                                                           | Replacement Value/Token                               |
+| --------------------------------------------------------------- | ----------------------------------------------------- |
+| `@include high-contrast-outline`                                | `outline: var(--p-border-width-1) solid transparent;` |
+| `@include high-contrast-outline($border-width: <border-width>)` | `outline: <border-width> solid transparent;`          |
+
+For `<border-width>` instances that are functions, see the [`border-width()`](#border-width) section for repalcing it.
```
Comment (alex-page) [inline documentation/guides/migrating-from-v8-to-v9.md]:
```suggestion
For `<border-width>` instances that are functions, see the [`border-width()`](#border-width) section for replacing it.
```

**Review by alex-page — APPROVED** [review]

File: documentation/guides/migrating-from-v8-to-v9.md [hunk 3]
```diff
@@ -121,6 +121,8 @@ A list of functions/mixins and their value equivalents or new token values.
 | `border-radius()`<br>`border-radius(base)` | `--p-border-radius-base`  |
 | `border-radius(large)`                     | `--p-border-radius-large` |
 
+#### `border-width()`
```
Comment (lgriffee) [inline documentation/guides/migrating-from-v8-to-v9.md:124]: Adding this because I'm linking to the header below.

**Review by aveline — APPROVED** [review]
